enginetest/queries.go

```diff
@@ -8418,6 +8430,14 @@ var errorQueries = []QueryErrorTest{
 		Query:       `CREATE FULLTEXT INDEX idx ON opening_lines(opening_line)`,
 		ExpectedErr: sql.ErrUnsupportedFeature,
 	},
+	{
+		Query:       `SELECT * FROM datetime_table where date_col >= 'not a valid date'`,
```

Comment:
YES! This makes me very happy. Silently converting to nil and returning incorrect results was super dangerous, so I'm very happy we have tests for this now and return an error instead.